lms/djangoapps/courseware/plugins.py

```diff
@@ -161,3 +162,51 @@ def get_allowed_operations(cls, course_key: CourseKey, user: Optional[User] = No
             # There is nothing to configure for calculator yet.
             "configure": False,
         }
+
+class ProctoringCourseApp(CourseApp):
+    """
+    Course App config for proctoring app.
+    """
+
+    app_id = "proctoring"
+    name = _("Proctoring")
+    description = _("Maintain exam integrity by enabling a proctoring solution for your course")
+
+    @classmethod
+    def is_available(cls, course_key: CourseKey) -> bool:
+        """
+        Proctoring is available for all courses.
+        """
+        return True
```

Comment:
I think this should check against the `ENABLE_SPECIAL_EXAMS` feature toggle, similar to what the textbook app is doing above

https://github.com/edx/edx-platform/blob/af82c6270ac4b7899f9d80005dbc603ea9730f49/cms/envs/common.py#L312